src/02-common-principles.md

```diff
@@ -48,9 +48,15 @@ misunderstanding we clarify them here.
     subject performed multiple conceptually different behaviors (with different
     sets of instructions) they will be considered one (combined) task.
 
-1.  **Event** - a stimulus or subject response recorded during a task. Each
-    event has an onset time and duration. Note that not all tasks will have
-    recorded events (e.g., “resting state”).
+1.  **Event** - something that happens or may be perceived as happening at a particular
+    moment during the recording.
+    Events are most commonly associated with sudden-onset sensory stimulus presentations
+    or with the distinct onset of a subject motor action.
+    Other events may include unplanned incidents,
+    changes in task instructions or experiment control parameters,
+    and noted data feature occurrences.
+    In BIDS, each event has an onset time and duration.
+    Note that not all tasks will have recorded events (e.g., "resting state").
```

Comment:
```suggestion
    Note that not all tasks will have recorded events (e.g., continuous recordings of "resting state" activity).
```

Reply:
I updated your suggestion to exclude "continuous" from "continuous recordings of resting state activity" ... because I think it's an unneeded constraint